image/base/Dockerfile

```diff
@@ -29,10 +29,12 @@ RUN sh -c "fping proxy && echo 'Acquire { Retries \"0\"; HTTP { Proxy \"http://p
 RUN apt-mark hold initscripts
 RUN apt-get -y upgrade
 
-RUN DEBIAN_FRONTEND=noninteractive apt-get install -y locales locales-all
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y locales
 ENV LC_ALL en_US.UTF-8
 ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US.UTF-8
+ADD locale.gen /etc/locale.gen
```

Comment:
I don't think we should maintain a giant list of locales since we need to keep what is uncommented in the list in-sync with the `LC_ALL`, `LANG` and `LANGUAGE` ENVs. I think we should just do something like `echo 'en_US.UTF-8 UTF-8' >> /etc/locale.gen` which is what the [official postgres docker image do](https://github.com/docker-library/postgres/blob/c9906f922daaacdfc425b3b918e7644a8722290d/16/bookworm/Dockerfile#L61).

Reply:
A `sed` to uncomment the LANG entry seemed like the only reliable way to convert a LANG value to its corresponding entry in /etc/locale.gen.

How should we update instead when users set `LANG=es_ES.UTF-8`, (requiring an entry of `es_ES.UTF-8 UTF-8`) or other non-UTF-8 locales such as `LANG=es_ES@euro` (requiring an entry of `es_ES@euro ISO-8859-15`)?

Reply:
....Oh, it's in `/usr/share/i18n/SUPPORTED` per the header comment, isn't it 🤦